pilot/proxy/envoy/mixer.go

```diff
@@ -145,7 +145,7 @@ func buildMixerOpaqueConfig(check, forward bool, destinationService string) map[
 
 // Mixer filter uses outbound configuration by default (forward attributes,
 // but not invoke check calls)
-func mixerHTTPRouteConfig(role proxy.Node, instances []*model.ServiceInstance, config model.IstioConfigStore) *FilterMixerConfig { // nolint: lll
+func mixerHTTPRouteConfig(role proxy.Node, instances []*model.ServiceInstance, outboundRoute bool, config model.IstioConfigStore) *FilterMixerConfig { // nolint: lll
```

Comment:
Will this approach work for Ingress proxy and Egress proxy.

As I understand,  we need to call Mixer for Ingress and Egress proxy?   You can double check with @kyessenov and @mandarjog